src/Sql/SqlBase.php

```diff
@@ -492,12 +492,21 @@ public function scheme()
     /**
      * Extract the name of all existing tables in the given database.
      *
-     * @param bool $quoted
-     *  If TRUE, each table name should be appropriately quoted for the RDMS.
      * @return array|null
      *   An array of table names which exist in the current database.
      */
-    public function listTables($quoted = false)
+    public function listTables()
+    {
+    }
+
+    /**
+     * Extract the name of all existing tables in the given database.
+     *
+     * @return array|null
+     *   An array of table names which exist in the current database,
+     *   appropriately quoted for the RDMS.
+     */
+    public function listTablesQuoted()
```

Comment:
The base implementation could call through to `listTables()`. That way contrib drivers that did not implement this method would not break.

Reply:
Thanks, I had a feeling I was missing something :)